### PR TITLE
Support CGN and new bonus items visibility handling logic

### PR DESCRIPTION
**Pull request by CrisTofani**

### CODE REVIEW

File: bonus/bonusAvailable.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 ### bonus_available_v2.json
 - pointed by app > `1.10.0.4`
-
-
-
+- A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```
Comment (Undermaken) [inline bonus/bonusAvailable.md:18]: Could you talk also about `hidden` property that will be deprecated?

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: bonus/bonusAvailable.md (hunk 1 above)
Comment (CrisTofani) [reply bonus/bonusAvailable.md:18]: Added in 78dbe19

File: bonus/bonusAvailable.md [hunk 2]
```diff
@@ -15,6 +15,9 @@
 
 ### bonus_available_v2.json
 - pointed by app > `1.10.0.4`
+- A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```
Comment (Undermaken) [inline bonus/bonusAvailable.md:18]:
```suggestion
- '> 1.18.0.3' A new logic for handling a bonus visibility has been implemented. Now every bonus has a new property `visibility` which can be evaluated with 3 different values:
```
What do you think if we write the last version that implement the old logic?
in this way we can avoid **WIP** in other points

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: bonus/bonusAvailable.md (hunk 2 above)
Comment (CrisTofani) [reply bonus/bonusAvailable.md:18]: I was actually waiting for the latest version of the app to be released 😄 now I can update readme information!

**Review by Undermaken — APPROVED** [review]

**Pull request merged**